app/src/main/java/de/westnordost/streetcomplete/quests/max_speed/AddMaxSpeedForm.kt

```diff
@@ -48,11 +48,9 @@ class AddMaxSpeedForm : AbstractQuestFormAnswerFragment<MaxSpeedAnswer>() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        val couldBeSlowZone = countryInfo.isSlowZoneKnown && POSSIBLY_SLOWZONE_ROADS.contains(osmElement!!.tags["highway"]!!)
-        zone.visibility = if (couldBeSlowZone) View.VISIBLE else View.GONE
+        zone.isGone = !countryInfo.isSlowZoneKnown || osmElement!!.tags["highway"]!! !in POSSIBLY_SLOWZONE_ROADS
```

Comment:
Mh, I find it better to have the `couldBeSlowZone` variable because it explains what the checks are made for. When assigning directly to `isGone`, it's a double negation and more difficult to understand.

The zone choice should only be shown if the street in question could be a slow zone.